yt/odin/checks/bin/queue_agent_alerts/__main__.py

```diff
@@ -13,7 +13,8 @@ def run_check(secrets, yt_client, logger, options, states):
 
     generic_alerts_by_stage_cluster = defaultdict(lambda: defaultdict(dict))
 
-    for queue_agent_stage_cluster in queue_agent_stage_clusters:
+    # assuming options["queue_agent_stage_clusters"] is a dict {cluster_name: proxy}
+    for queue_agent_stage_cluster, proxy in queue_agent_stage_clusters.items():
```

Comment:
It is a little bit weird that for cluster name you add prefix `queue_agent_state`, but for proxy do no add